101-batch-with-keyvault-pfx-password/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,74 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "certificatePrefixName": {
+      "type": "string"
+    },
+    "certificatePfxBase64": {
+      "type": "securestring"
+    },
+    "certificatePfxPassword": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "storagename": "[concat('storage', uniqueString(resourceGroup().id))]",
+    "batchname": "[concat('batch', uniqueString(resourceGroup().id))]",
+    "locationname": "[resourceGroup().location]",
+    "keyvaultname": "[concat('keyvault', uniqueString(resourceGroup().id))]",
+    "tenantId": "[subscription().tenantId]",
+    "certificateName": "[concat(parameters('certificatePrefixName'), uniqueString(resourceGroup().id))]"
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-08-01",
+      "name": "[variables('certificateName')]",
```

Comment:
Why are we creating this?  We don't seem to use it, but just using the cert CI already has available?

Reply:
Matching thumbprint for GEN-SELFSIGNED-CERT-PFXDATA is not given in contribution guide and it is not matching with other thumbprint for ex. GEN-SF-CERT-THUMBPRINT. 
Not sure but I think it would be very helpful if selfsigned thumbprint given in doc.

GEN-SELFSIGNED-CERT-PFXDATA - PFX data for a sample self signed cert
GEN-SELFSIGNED-CERT-CERDATA - CER data for a sample self signed cert
GEN-SELFSIGNED-CERT-PASSWORD - password for a sample self signed cert
GEN-SELFSIGNED-CERT-DNSNAME - DNS name for a sample self signed cert